oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb

```diff
@@ -21,7 +21,7 @@
 
   include Pedant::RSpec::CookbookUtil
 
-  context "DELETE /cookbooks/<name>/<version>" do
+  context "DELETE /cookbook_artifacts/<name>/<version>" do
```

Comment:
@randomcamel I'm fine leaving that call to you or @danielsdeleo who know this code better than I most likely.